api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -144,7 +144,6 @@ private OpenmrsUtil() {
 	 * @return [List toAdd, List toDelete] with respect to origList
 	 */
 	public static <E> Collection<Collection<E>> compareLists(Collection<E> origList, Collection<E> newList) {
-		// TODO finish function
 		
 		Collection<Collection<E>> returnList = new ArrayList<>();
```

Comment:
thanks @settix-dev  you can remove this space too!!

Reply:
I have removed the space too.